deploy/manifests/base/kustomization.yaml

```diff
@@ -5,3 +5,4 @@ resources:
 - service.yaml
 - validation-service.yaml
 - kong-ingress-dbless.yaml
+- monitor.yaml
```

Comment:
We should place this in the kustomization at `config/base/kustomization.yaml` for now. Technically that's the proper location for 2.x-specific content we add on top of the 1.x manifests under `deploy/manifests/base`. That's kind of a moot point with the removal of 1.x code and tooling from next, but it may matter if we need to cherry-pick some fix for the original manifests into the 1x branch pending cleanup of the 1.x content still present in next.

There's also a `prometheus` dir under `config` that we should use (with the kustomization.yaml including `- prometheus` instead of `monitor.yaml` in case we add more content there later). The monitor.yaml there can be overwritten entirely with the new PodMonitor; it's just a default config that we've never actually used and don't need.

Reply:
changed accordingly.